src/functions-and-arrays.js

```diff
@@ -51,15 +90,28 @@ const wordsUnique = [
   'simple',
   'bring'
 ];
-
-function uniquifyArray() {}
+  function uniquifyArray(wordsUnique) {
+    if (wordsUnique.length === 0 ) {
+      return null }
+    return wordsUnique.filter((item, index) => wordsUnique.indexOf(item) === index)
+  }
 
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(wordsFind, word5) {
```

Comment:
here you need to write for loop,inside if..else statement  :
 function doesWordExist(wordsFind, word5) {
  if (wordsFind.length === 0) {
    return null
  }
  else {
    for (i = 0; i < wordsFind.length; i++) {
      if (wordsFind[i] == word5) {
        return true
      } 
    }
    return false
  }
}